hw7/Task_3.ts

```diff
@@ -0,0 +1,10 @@
+// Task 3
+// Дан массив:
+//     ```javascript
+//     const numbers = [5, 43, 63, 23, 90]
+// ```
+// Удалите все элементы в массиве и выведите в консоль полученный результат.
+
+const numToDelete: number [] = [5, 43, 63, 23, 90]
+numToDelete.splice(0, numToDelete.length)
+console.log(numToDelete)
```

Comment:
<img width="802" alt="image" src="https://user-images.githubusercontent.com/47811445/221351709-7127ac70-40d6-4ad3-8065-b8432ec1f83a.png">
incorrect result

Reply:
Cannot get why incorrect. Console.log shows that the array is empty now.

Reply:
Indeed, I probably misread the task :)